src/plugins/controls/public/options_list/components/options_list_editor_options.tsx

```diff
@@ -39,6 +41,16 @@ export const OptionsListEditorOptions = ({
           }}
         />
       </EuiFormRow>
+      <EuiFormRow display="columnCompressedSwitch">
+        <EuiSwitch
+          label={OptionsListStrings.editor.getAllowExclude()}
+          checked={!state.allowExclude}
+          onChange={() => {
+            onChange({ allowExclude: !state.allowExclude });
+            setState((s) => ({ ...s, allowExclude: !s.allowExclude }));
+          }}
+        />
+      </EuiFormRow>
```

Comment:
This adds a toggle that allows authors to disable the option of allowing users to exclude their selections.

https://user-images.githubusercontent.com/8698078/197568216-2a4a5865-edfe-4067-8371-5f15bfcfda4f.mov